deployment/cloud-run/test_routing_debug.py

```
@@ -5,80 +5,130 @@
+        for endpoint, rule in endpoints.items():
+            print(f"{endpoint} -> {rule}")
+
+        # Check what Flask-RESTX created for the root route
+        print("\n=== Flask-RESTX root route details ===")
+        for rule in self.app.url_map.iter_rules():
+            if rule.rule == '/':
+                print(f"Root route: {rule.rule} -> {rule.endpoint}")
+                print(f"  Methods: {rule.methods}")
+                print(f"  View function: {rule.endpoint}")
+
+    def test_routing_71(self):
```

Ruff (0.12.2)

Missing docstring in public method

(D102)